polaris.shopify.com/pages/examples/ExampleStyleGuideComponents/AlignBlockComponent/Placeholder.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {Text, Inline} from '@shopify/polaris';
+
+type Align = 'start' | 'end' | 'center';
+
+export interface PlaceholderProps {
+  label?: string;
+  height?: string;
+  width?: string;
+  childAlign?: Align;
+}
+
+export const Placeholder = ({
```

Comment:
There's no default export so `yarn build` fails.
You can change this line to
```suggestion
const Placeholder = ({
```

and at the end of the file, do `export default Placeholder`.
You'll also have to update wherever this is imported so that its:
`import Placeholder from './xyz'` instead of `import {Placeholder} from './xyz'`

This applies to `AlignBlockComponent/Placeholder.tsx`, `DefaultBlockComponent/Placeholder.tsx`, and `SpacingBackgroundExample.tsx` and wherever they're being imported.